src/Slot-Examples/InitializedClassVariable.class.st

```diff
@@ -15,7 +15,7 @@ Class {
 
 { #category : #'class building' }
 InitializedClassVariable >> installingIn: aClass [
-	isBlock 
-		ifTrue: [ self write: (default copy cull: aClass)]
-		ifFalse: [ self write: default]
+	default isBlock 
+		ifTrue: [ self write: (default cull: aClass)]
+		ifFalse: [ self write: default copy]
```

Comment:
Why the #copy is required here?

Reply:
maybe it is not needed... we could make the default read only instead. I just wanted to not have people change the default if it is a string...